### PR TITLE
888728: Prepare the UG documentation for SfDataGrid scrolling performance.

### PR DESCRIPTION
**Pull request by RasikaPalanisamy**
Prepared the UG documentation for SfDataGrid scrolling performance.

### CODE REVIEW

File: MAUI/DataGrid/performance.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Performance in MAUI DataGrid (SfDataGrid)
+
+## LoadUIView
+
+By default, the `UIElement` is loaded inside the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html). The `DataGridColumn.LoadUIView` property determines whether to load a `UIElement` inside the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) or to draw the cell value directly on the `canvas` of the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) on the Android platform.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/performance.md]: Changes as per the concerns provided.

File: MAUI/DataGrid/performance.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+If `LoadUIView` is set to `false`, the cell value of the column will be drawn directly on the `canvas` of the grid cells to enhance performance. However, when `LoadUIView` is set to `true`, a `UIElement` ([SfDataGridLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGridLabel.html)) will be loaded in the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html).
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/performance.md]: Changed as per the concerns provided.

File: MAUI/DataGrid/performance.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The default value of `LoadUIView` is `True`.  The scrolling performance improved when `LoadUIView` was set to `False`.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/performance.md]: Changed as per the concerns provided.

File: MAUI/DataGrid/performance.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+column.LoadUIView = false;
+dataGrid.Columns.Add(column);
+
+{% endhighlight %}
+
+You can also download the entire source code of this from [GitHub](https://github.com/SyncfusionExamples/How-to-improve-scrolling-performance-in-.NET-MAUI-DataGrid).
+
+### Limitations
+
+* Implicit and explicit padding is not supported. [CellPadding](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_CellPadding) is applied to the left and right, but the top and bottom [CellPadding](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_CellPadding) of [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) are determined by the content.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/performance.md]: Changed as per the concerns provided.

File: MAUI/DataGrid/performance.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+* LineBreakMode is not supported.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/performance.md]: Changed as per the concerns provided.

File: MAUI/DataGrid/performance.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+* Accessibility does not work.
+
+* Runtime theme switching is not supported.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/performance.md]: Changed as per the concerns provided.

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine.

**Review by FarjanaParveen — APPROVED** [review]
This is fine

**Pull request merged**